internal/tools/aws/route53.go

```diff
@@ -546,7 +546,7 @@ func prettyRoute53Response(resp *route53.ChangeResourceRecordSetsOutput) string
 
 // parseHostedZoneResourceID removes prefix from hosted zone ID.
 func parseHostedZoneResourceID(hostedZone *route53.HostedZone) (string, error) {
-	id := strings.TrimLeft(*hostedZone.Id, hostedZonePrefix)
+	id := strings.TrimLeft(*hostedZone.Id, hostedZonePrefix[0:len(hostedZonePrefix)-1])
```

Comment:
Did you find a bug here or was this just a general improvement?

Reply:
Linter complained because `TrimLeft` uses the second argument as a character list to remove from the first argument, but `hostedZonePrefix` starts and ends with a slash.

Reply:
Hmm, yeah I think either would work, but this seems fine to me as is.